### DISCUSSION TITLE
Consumption_heatpump value with external MQTT message problems

### DISCUSSION THREAD

**Discussion by nash1975** [Arnold-n/P1P2MQTT#100] [Q&A]
First of all, many many thanks for this great update.
It makes a lot of things much easier for controlling my daikin E-Series heatpump.

I have 1 issue right now.

I would like to import MQTT messages from my external power meter.
I managed my related ESP-home device to publish under the right topic.
(See attached screenshot)
But the "consumption_heatpump" entity is still showing zero.
Any thoughts?

![2024-06-04 14_53_16-MQTT Explorer](https://github.com/Arnold-n/P1P2MQTT/assets/132844511/933959d0-2fde-4b81-a3ed-f66e21cec836)

**Comment by Arnold-n** [comment]
Your're welcome! And sorry for the docu bug, I just fixed the topic in the documentation: the correct topic is `P1P2/P/meter/U/9/Electricity_Power`.

There is also a time-out of 60s, so the power consumption must be (re)published at least every 60s.

**Reply by Arnold-n** [reply]
Alternatively you can change the topic listened to, via telnet or P1P2/W with command `P35 P1P2/P/meter/bridge0/Electricity_Power` followed by `D5` to save and `D4` or `D0` to resubscribe.

**Comment by nash1975** [comment]
Seems to work now.
But I directly ran into this timeout issue.
But this I can fix on my client device via ESPhome.

Thanks for the quick response

**Comment by Arnold-n** [comment]
Great, thanks for the feedback.
